app/angular/src/server/framework-preset-angular-ivy.ts

```diff
@@ -0,0 +1,53 @@
+import { Configuration } from 'webpack';
+import { process as ngccProcess } from '@angular/compiler-cli/ngcc';
+import * as path from 'path';
+import { Options } from './framework-preset-angular-cli';
+
+/**
+ * Run ngcc for converting modules to ivy format before starting storybook
+ * This step is needed in order to support Ivy in storybook
+ *
+ * Information about Ivy can be found here https://angular.io/guide/ivy
+ */
+export const runNgcc = () => {
+  ngccProcess({
+    // should be async: true but does not work due to
```

Comment:
Shouldn't we change this or should we wait ?

Reply:
It does work without being async. The downside is if someone starts the user app + storybook at the same time and both start `ngcc` then storybook will most likely crash 

So it's an edge case but it also does not work yet to set it to async